data/tekton-pipelines/windows-bios-installer-pipeline.yaml

```diff
@@ -0,0 +1,273 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines
+roleRef:
+  kind: ClusterRole
+  name: windows-pipelines
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines
+  namespace: kubevirt-os-images
+roleRef:
+  kind: ClusterRole
+  name: modify-data-object-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: modify-data-object-task
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipeline
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: windows-pipelines
+rules:
+  - verbs:
+      - create
+    apiGroups:
+      - ""
+    resources:
+      - pods
+  - verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - delete
+    apiGroups:
+      - kubevirt.io
+    resources:
+      - virtualmachines
+      - virtualmachineinstances
+  - verbs:
+      - '*'
+    apiGroups:
+      - kubevirt.io
+    resources:
+      - virtualmachines/finalizers
+  - verbs:
+      - '*'
+    apiGroups:
+      - ''
+    resources:
+      - persistentvolumeclaims
+  - verbs:
+      - '*'
+    apiGroups:
+      - cdi.kubevirt.io
+    resources:
+      - datavolumes
+  - verbs:
+      - update
+    apiGroups:
+      - subresources.kubevirt.io
+    resources:
+      - virtualmachines/start
+      - virtualmachines/stop
+      - virtualmachines/restart
+
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: windows-bios-installer
+spec:
+  params:
+    - name: winImageDownloadURL
+      description: Download URL to Windows installation ISO (English United States x64 version is needed). You can follow e.g. https://www.microsoft.com/en-us/software-download/windows10ISO to get windows 10 iso.
+      type: string
+    - name: autounattendConfigMapName
+      description: Name of the ConfigMap containing the sysprep configuration files (autounattend.xml, etc.).
+      type: string
+      default: windows10-bios-autounattend
+    - name: virtioContainerDiskName
+      description: Reference to the containerdisk containing the virtio-win drivers ISO.
+      type: string
+      default: quay.io/kubevirt/virtio-container-disk:latest
```

Comment:
It can't because the value is replaced by https://github.com/kubevirt/ssp-operator/blob/master/internal/operands/tekton-pipelines/tekton-pipelines.go#L161